src/app/[locale]/specs/account/page.mdx

```diff
@@ -0,0 +1,90 @@
+
+The deactivation and suspension states are implied to be temporary, though they might have indefinite time periods. Deletion and takedowns are implied to be more final, but can still technically be reverted (and frequently are, in practice).
+
+Downstream services might decide to delete content rapidly upon takedown (like deletion), or simply stop redistributing content until a later time.
+
+Identities might also be tombstoned, which is also implied to be permanent and final, but can technically be reversed in certain conditions. If the account's identity is `tombstoned`, the account hosting status should be interpreted as `deleted`.
+
+To summarize, when an account status is non-`active`, the content that hosts should not redistributed includes:
+
+- repository exports (CAR files)
+- repo records
+- transformed records ("views", embeds, etc)
+- blobs
+- transformed blobs (thumbnails, etc)
+
+### Hosting Status Propagation
+
+Unlike other aspects of atproto, account status is not self-certifying, and there is not a method for authenticated transfer or redistributing of account status between arbitrary parties.
+
+Most commonly, account status is propagated “hop by hop”, with each “downstream” service accepting and adopting the hosting status for accounts at their upstream. If an intermediate service (such as a relay) overrides the status of their upstream (for example, an infrastructure account takedown), they will broadcast and propagate that action downstream to subscribers.
+
+By default, if an account’s current active PDS is not available, account status should remain unchanged, at least for some time period. This keeps the network resilient and accounts online during temporary infrastructure outages. Services may decide their own policies for accounts whose PDS hosts remain offline for long periods of time.
+
+The general expectation is that downstream services will not list an account as `active` if it is inactive at an upstream service. However, the concept of “upstream” is informal, and services may need to define their own policies for some corner-cases. For example, an AppView might consume from multiple independent relay providers, who report differing statuses for the same account, due to policy differences. If the network connection between a relay and PDS is disruptive, they may report different statuses for the same account. 
+
+If there is doubt about about an account’s general status in the network, the account’s current active PDS host can be queried using the `com.atproto.sync.getRepoStatus` endpoint. If the account is not `active` there, it is generally expected to not be active elsewhere, especially in the case of user-initiated actions (deactivation or deletion).
+
+### Content Deletion
+
+Separate from overall account status, individual pieces of content may be removed. When accounts remove records from their public repositories, hosting services should remove public access to that content.
```

Comment:
Would this be crisper if it explicitly said the authoritative way of knowing whether an account has "removed records from its public repos"? i.e., if you (as an intermediate service or consumer) check the current PDS host for a piece of content you got elsewhere or earlier, and current host says 404, you SHOULD flush your cache and stop distributing it to others.

Reply:
The details around repositories and record deletion are mostly discussed in the Repository spec page and the Sync spec page (the later is in a concurrent PR right now). There is similar discussion in the Blobs spec page. This document ("Account Hosting") isn't about how to find out if a record has been deleted from a repo (that comes over the firehose, FWIW), it is about expectations for when that deletion event is received.

I guess it fits in a bit weird here, but it felt strange to talk so much about account-wide state changes without at least some mention of more granular content deletion at the same time.